source/expressions.tex

```diff
@@ -1509,7 +1509,7 @@
   C().C::~C();      // undefined behavior: temporary of type \tcode{C} destroyed twice
   using T = int;
   0 .T::~T();       // OK, no effect
-  0.T::~T();        // error: \tcode{0.} is a floating-point literal\iref{lex.fcon}
+  0.T::~T();        // error: \tcode{0.T} is a floating-point literal\iref{lex.fcon}
```

Comment:
No, "0.T" is not a floating-point literal; this is a literal with a _ud-suffix_.

Reply:
Do you agree it's a user-defined-floating-literal?

Reply:
Then, let's go with "error: 0.T is a user-defined-floating-literal". This seems to be the most precise of the choices.